CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## Breaking Changes
+
+* 2016-05-11: Upgrade of ElasticSearch from 1.7 to 2.3 [#449](https://github.com/etalab/udata/pull/449)
```

Comment:
What needs to be done to not break things? Upgrade ES at the same time? Run a migration?